src/Symfony/Component/HttpFoundation/Tests/CookieTest.php

```diff
@@ -151,4 +151,13 @@ public function testToString()
         $cookie = new Cookie('foo', 'bar', 0, '/', '');
         $this->assertEquals('foo=bar; path=/; httponly', $cookie->__toString());
     }
+
+    public function testRawCookie()
```

Comment:
Should we also add a test which creates a raw cookie for a request and then check if its contents are really not encoded?